src/languages.js

```diff
@@ -1,19 +1,21 @@
 import fs from 'fs'
+
 import { ERRORS } from './constants'
 
-export function getAvailableLanguages(dir) {
+export function getAvailableLanguages(directory, filesToIgnore = []) {
```

Comment:
What do you feel about changing these parameters to an object like `{ directory, exclude = [] }`? The `getAvailableLanguages` calls would become way more clearer